build.gradle

```diff
@@ -1,3 +1,3 @@
 plugins {
-    id 'com.jfrog.artifactory' version '4.18.2' apply false
+    id 'com.jfrog.artifactory' version '4.32.0' apply false
```

Comment:
is there somewhere you recommend to read about what a gradle conventions project is?

Reply:
created https://broadworkbench.atlassian.net/browse/TSPS-56 to make sure we deal with this issue as well as the issue below of having unused dependencies in our service gradle build file.  Don't think I will address those changes in this PR just to avoid scope creep.

Reply:
Ok, I missed the earlier message but these problems should be fixable, I can help when you're at that point. Maybe slack me too in case I miss the GitHub notification.

Reply:
will do!